common/lib/xmodule/xmodule/tests/test_html_module.py

```diff
@@ -24,6 +24,39 @@ def instantiate_descriptor(**field_data):
     )
 
 
+class HtmlModuleCourseApiTestCase(unittest.TestCase):
+    """
+    Ensure that student_view_data can handle likely input,
+    and doesn't modify the HTML in any way.
+    """
+    descriptor = Mock()
+
+    def prep_module(self, html):
```

Comment:
@bradmerlin nit: saving three keystrokes every time? :) `prepare_module` might be a better name. `_prepare_module` even better, since this method is inherently "private"